website/_dogfooding/_pages tests/code-block-tests.mdx

```diff
@@ -27,6 +27,41 @@ See:
 - https://github.com/facebook/docusaurus/pull/3749
 - https://github.com/facebook/docusaurus/pull/6177
 
+## Code block prism language tests
+
+Code block with/without the good prism language case(lower or upper) in `additionalLanguages[]`
+
+```php
+<?php
+$x=15;
+$y=30;
+$z=$x+$y;
+echo "Sum: ",$z;
+?>
+```
+
+```PHP
```

Comment:
test added, but unfortunately highlighting doesn't work yet

We probably need to lowercase the meta string language in the code block theme component too